Makefile

```diff
@@ -20,6 +20,18 @@ OBJS := $(SRCS:.cpp=.o)
 

```

Comment:
the --resolve tag in curl will let us use the correct host name without changing /etc/hosts
But if we want the browser to recognize hostname correctly I think we need /etc/hosts
If you don't want it, just let me know and I'll remove it ( or if there is a prettier way.. I just copied my inception makefile.. lol)